mobile/ios/scripts/build_frameworks.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh -e
 
+ROOTDIR=$(cd $(dirname $0) && pwd -P)
+cd "$ROOTDIR/.."
```

Comment:
I don't want to change directory in the scripts. Could you run the cd command in a subshell?